Improve explanations of figures 2 and 3. [skip ci] (#51)
input-output-hk/fm-ledger-rules: docs/delegation_design_spec/delegation_design_spec.tex

relation.

Figure~\ref{fig:relationship-keys-certificates} shows the relationships between
the different types of certificates and keys.
%
Dashed arrows represent relationships between the different keys: the owner of
a staking key delegates his staking rights to the owner of a stake pool cold
key, using a delegation certificate.
%

> The owner of an operational cold key grants staking rights to the owner of a

Reviewer: ```suggestion
The owner of a cold and hot key grants the staking rights of their cold key to their
```
